access-control-app/docs/api.md

```diff
@@ -45,13 +45,9 @@ This defines common headers on the Access Control API.
 
 Content-Type is a standard HTTP header that specifies the content type of the body of the request. Access Control only supports `application/json` as the representation type for data.
 
-#### <a name="echo-token-header"></a> Echo-Token Header
-
-All Access Control API operations require specifying a token obtained from URS or ECHO. The token should be specified using the `Echo-Token` header.
-
 #### <a name="authorization-header"></a> Authorization Header
 
-The token can alternatively be specified using the `Authorization: Bearer` header, and by specifying a Bearer token.
+All Access Control API operations require specifying a token obtained from Earthdata Login (EDL). The token should be specified using the `Authorization: Bearer` header followed by the EDL bearer token.  For more information on obtaining an EDL bearer token, please reference the documentation [here](https://wiki.earthdata.nasa.gov/display/EL/How+to+Generate+a+User+Token).
```

Comment:
Earthdata Login should be a direct link. I know (and read in the wiki) that a link can be revived there, but people are in a hurry.